app/models/application_qualification.rb

```diff
@@ -1,4 +1,5 @@
 class ApplicationQualification < ApplicationRecord
+  self.ignored_columns += %w[equivalency_details]
```

Comment:
Don't think you need this anymore, the column has been removed

Reply:
Thanks that was a rebase mistake I think